post_processing/remap/remap_helpers.jl

```diff
@@ -200,157 +130,14 @@ function remap2latlon(filein, data_dir, remap_tmpdir, weightfile, nlat, nlon)
     ᶠw = Geometry.WVector.(Y.f.u₃)
     ᶜw = ᶜinterp.(ᶠw)
     nc_w[:, 1] = ᶜw
-    # diagnostic variables
-    nc_pres[:, 1] = diag.pressure
-    nc_T[:, 1] = diag.temperature
-    nc_θ[:, 1] = diag.potential_temperature
-    nc_K[:, 1] = diag.kinetic_energy
-    nc_vort[:, 1] = diag.vorticity
-    nc_T_sfc[:, 1] = diag.sfc_temperature
-    nc_z_sfc[:, 1] = Fields.level(Fields.coordinate_field(Y.f.u₃).z, half)
-    nc_qt_sfc[:, 1] = diag.sfc_qt
-
-    if :ρq_tot in propertynames(Y.c)
-        nc_qt[:, 1] = Y.c.ρq_tot ./ Y.c.ρ
-        nc_RH[:, 1] = diag.relative_humidity
-        nc_cloudliq[:, 1] = diag.q_liq
-        nc_cloudice[:, 1] = diag.q_ice
-        nc_watervapor[:, 1] = diag.q_vap
-        nc_cloudfrac_gm[:, 1] = diag.cloud_fraction_gm
-    end
-
-    if :precipitation_removal in propertynames(diag)
-        nc_precipitation_removal[:, 1] = diag.precipitation_removal
-        nc_column_integrated_rain[:, 1] = diag.column_integrated_rain
-        nc_column_integrated_snow[:, 1] = diag.column_integrated_snow
-    end
-
-    if :sfc_flux_energy in propertynames(diag)
-        nc_sfc_flux_energy[:, 1] = diag.sfc_flux_energy
-        nc_sfc_flux_u[:, 1] = diag.sfc_flux_u
-        nc_sfc_flux_v[:, 1] = diag.sfc_flux_v
-        if :sfc_evaporation in propertynames(diag)
-            nc_sfc_evaporation[:, 1] = diag.sfc_evaporation
-        end
-    end
-
-    if :lw_flux_down in propertynames(diag)
-        nc_lw_flux_down[:, 1] = diag.lw_flux_down
-        nc_lw_flux_up[:, 1] = diag.lw_flux_up
-        nc_sw_flux_down[:, 1] = diag.sw_flux_down
-        nc_sw_flux_up[:, 1] = diag.sw_flux_up
-    end
-
-    if :clear_lw_flux_down in propertynames(diag)
-        nc_clear_lw_flux_down[:, 1] = diag.clear_lw_flux_down
-        nc_clear_lw_flux_up[:, 1] = diag.clear_lw_flux_up
-        nc_clear_sw_flux_down[:, 1] = diag.clear_sw_flux_down
-        nc_clear_sw_flux_up[:, 1] = diag.clear_sw_flux_up
-    end
 
-    if :draft_q_tot in propertynames(diag)
-        nc_draft_qt[:, 1] = diag.draft_q_tot
-        nc_draft_RH[:, 1] = diag.draft_relative_humidity
-        nc_draft_cloudliq[:, 1] = diag.draft_q_liq
-        nc_draft_cloudice[:, 1] = diag.draft_q_ice
-        nc_draft_watervapor[:, 1] = diag.draft_q_vap
-        nc_draft_cloudfrac[:, 1] = diag.draft_cloud_fraction
-        nc_draft_area[:, 1] = diag.draft_area
-        nc_cloudfrac[:, 1] = diag.cloud_fraction
-        nc_tke[:, 1] = diag.env_tke
-        nc_mixing_length[:, 1] = diag.env_mixing_length
-    end
     close(nc)
 
-
     datafile_latlon =
         joinpath(out_dir, first(splitext(basename(filein))) * ".nc")
-    dry_variables = [
-        "rho",
-        "e_tot",
-        "u",
-        "v",
-        "w",
-        "pressure",
-        "temperature",
-        "potential_temperature",
-        "kinetic_energy",
-        "vorticity",
-        "sfc_temperature",
-        "sfc_elevation",
-        "sfc_qt",
-    ]
-    if :ρq_tot in propertynames(Y.c)
```

Comment:
I think I might be missing this in the newer ClimaAnalysis system - where in the `remap_helper` modifications do we account for fields in the state `Y.c` that correspond to moisture ? Dry variables are still retained the same as in the original code iiuc.